source/js/Module/Event/Components/AgeFilter.js

```diff
@@ -1,32 +1,61 @@
-import { Dropdown } from '@helsingborg-stad/hbg-react';
-import PropTypes from 'prop-types';
-
-const AgeFilter = ({ translation, ageRange, onAgeChange }) => (
-  <div>
-    <Dropdown title={translation.selectAge}>
-      {ageRange.map(item => (
-        <label key={item.value} className="checkbox u-px-1">
-          <input
-            type="checkbox"
-            value={item.value}
-            onChange={e => onAgeChange(e, item.value)}
-            checked={item.checked}
-          />{' '}
-          {item.value} {translation.years}
-        </label>
-      ))}
-    </Dropdown>
-  </div>
-);
-
-AgeFilter.propTypes = {
-  ageRange: PropTypes.array.isRequired,
-  onAgeChange: PropTypes.func.isRequired,
-  translation: PropTypes.object,
-};
+import React from "react";
+import { useState, useEffect } from "react";
+import { Input } from "@helsingborg-stad/hbg-react";
+
+const AgeFilter = ({ translation, ageRange, onAgeRangeChange, minValue, maxValue, onChange }) => {
+    
+  const minLimit = 0;
+  const maxLimit = 100;
+
+  const minChanged = (e) => {
+    const isMoreThenMax = parseInt(e.target.value) >= maxValue;
+    const isLessThanMinLimit = minLimit > parseInt(e.target.value);
+
+    const value = isMoreThenMax ? maxValue - 1 : isLessThanMinLimit ? minLimit : parseInt(e.target.value);
+
+    onChange({
+      min: value,
+      max: maxValue,
+    });
+  }
+
+  const maxChanged = (e) => {
+    const isLessOrEqualsMin = parseInt(e.target.value) <= minValue;
+    const isMoreThenMaxLimit = parseInt(e.target.value) > maxLimit;
+
+    const value = isLessOrEqualsMin ? minValue + 1 : isMoreThenMaxLimit ? minValue +1 : parseInt(e.target.value);
```

Comment:
parseInt is adding a lot noise. perhaps it will look better if we define `const value = parseInt(e.target.value)` at the top of the function and move the online to {max: isLessOrEqualsMin ? etc...}